### DISCUSSION TITLE
are there any incompatabiltys with kontraption?

### DISCUSSION THREAD

**Discussion by aaaaaa1233** [illucc/Kontraption#30] [General]
i use modernfix, ferrite core and lazyDFU, yet it seems the connected textures are not connecting.

**Comment by aaaaaa1233** [comment]
im wondering what the incompatibilitys are so i dont get my macbook air fried or get wronged by using some incompatible mods.

**Comment by AngusAU293** [comment]
I'm pretty sure that mods like Pehkui are incompatible with VS2 in general.
